spec/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  extends: "airbnb",
+  env: {
+    jasmine: true,
+  },
+  plugins: [
```

Comment:
Do we need these here and in the top level file? Perhaps we can import the top level one and extend that?